README.md

```diff
@@ -3,91 +3,51 @@
+      `.env` file.
+   1. Generate a private key in the "Private keys" section. Download the
+      generated file, move it to your local repository root folder, rename it to
+      `towtruck.private-key.pem` (per the `.gitignore` file), then run
+      `script/encode-key towtruck.private-key.pem` from the repository root
+      folder. Copy the output of this command as the private key in your local
+      `.env` file.
 
-Once the app is registered, it should be installed to an account to allow
-Towtruck to track it. GitHub have instructions to do this
-[here](https://docs.github.com/en/apps/using-github-apps/installing-your-own-github-app).
+   > The `REDIRECT_URL_BASE`, `CLIENT_SECRET`, and `WEBHOOK_SECRET` can remain
+   > as their default values.
 
-##### Permissions
-
-Towtruck is still in early development so the exact set of needed permissions
-has not been finalised.
-
-##### Webhooks
-
-Towtruck is still in early development so the exact set of needed webhooks has
-not been finalised.
-
-#### Using the dxw Github App
-
-This will use the live data from dxw's repos. Any testing to configuration
-should be performed using a personal test app first. The environment variables
-are available here:
-<https://github.com/organizations/dxw/settings/apps/dxw-towtruck>. You may need
-to ask permission from an owner of the dxw organisation.
-
-### Environment variables
-
-In order for Towtruck to communicate with the GitHub API, it needs several
-pieces of information, configured through environment variables:
+### Seeding
 
-- `APP_ID`: The unique numeric ID assigned to the GitHub App. This can be found
-  in the dxw-towtruck app settings.
-- `PRIVATE_KEY`: The private key used to sign access token requests. Towtruck
-  expects this to be a `.pem` file generated by GitHub in the app settings and
-  encoded using Base64. This can be generated from the `.pem` file by running
-  `script/encode-key <path-to-pem-file>` in the terminal and copying the output.
-- `CLIENT_ID`: A unique alphanumeric ID assigned to the GitHub App. This can be
-  found in the dxw-towtruck app settings.
-- `CLIENT_SECRET`: A token used to authenticate API requests. These are
-  generated by GitHub in the dxw-towtruck app settings.
-- `WEBHOOK_SECRET`: A user-defined secret used to authenticate GitHub to
-  Towtruck for receiving webhooks. This must be exactly the same as it is
-  entered in the app settings on GitHub.
+Run `script/seed` to seed the database using live data from dxw's repositories.
+You can also seed the databse as a final step while bootstrapping using
+`script/bootstrap --seed`.
 
-### Seeding
+> This will call the Github API. Requests to this API are rate limited so take
+> care not to to run the script too often.
 
-Once all the other setup steps have been completed run `script/seed` or
-`script/bootstrap --seed` to seed the data. This will call the Github API which
-is rate limit so take care not to to run the script too often.
```

Comment:
I think it's worth keeping this information because Towtruck being open source means that it's possible that others may want to deploy their own instance and this documents that. Perhaps the headings/preamble should be updated to make it clearer that our instance is for our stuff and that other people will need to deploy their own instance?

Reply:
Also curious how the multi-organisation feature will work, but that's a separate conversation!

Reply:
> Is it specifically the stuff about setting up and using a new GitHub app that you were meaning here?

I think we should be documenting that, but also the set of permissions and webhooks that we use, as access to these needs to be given to the app in the settings (although I admit we're just using a "TBD" placeholder for these sections at the moment). I think it's also worth having an explanation of what the environment variables do rather than just how to change them, as well.

> Could probably reword the second prerequisite and then refer people to a separate section or a separate doc about setting up a GitHub app if using Towtruck for a different organisation

This might be a useful approach to take, so that we can keep the readme to the essentials to getting set up locally.